### PR TITLE
Update en.json

### PR DESCRIPTION
**Pull request by nethip**
Updating english Strings as per @peterflynn 's comments.

### CODE REVIEW

File: updates/stable/en.json [hunk 1]
```diff
@@ -11,15 +11,15 @@
                 "description": "Select any text block in the editor and fold markers will appear to collapse the selected text."
             },
             {
-                "name": "Improved Stability and Instant Search",
-                "description": "Major stability and performance improvements. Find in Files can now index and search robustly in thousands of files instantly."
+                "name": "Instant Search: Improved Stability and Performance",
+                "description": "Find in Files is now more stable as Brackets can now index and search robustly in thousands of files instantly."
```
Comment (peterflynn) [inline updates/stable/en.json]: Fwiw, I liked the original description here (only the heading was confusing).  The updated description doesn't read nearly as smoothly to me.

**Comment by peterflynn** [comment]
@nethip Lgtm, if you just revert that one description change.

**Comment by nethip** [comment]
@peterflynn Reverted the change.PR is up for review again,.

**Comment by peterflynn** [comment]
Looks good -- thanks!

**Pull request merged**